package.json

```diff
@@ -87,6 +86,7 @@
 		"module-alias": "^2.2.3",
 		"moment": "^2.29.4",
 		"node-ical": "0.16.1",
+		"pm2": "https://github.com/bugsounet/pm2#build",
```

Comment:
Why is the official package not used here?

Reply:
it's a draft... I test something but i plan to use official packet
I'm working about something else actually. I will see after ;)
Don't worry